src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -135,16 +135,19 @@
           current-public-key (re-frame/subscribe [:get-current-public-key])]
       [react/view {:style styles/messages-view}
        [react/scroll-view {:scrollEventThrottle    16
+                           :headerHeight styles/messages-list-vertical-padding
+                           :footerWidth styles/messages-list-vertical-padding
+                           :enableArrayScrollingOptimization true
```

Comment:
`:enableArrayScrollingOptimization` option is introduced into react-native-desktop port to enable rendering optimization for lists of items inside ScrollView (items dynamically mounted and unmounted from the scene graph)